src/panels/config/users/ha-user-editor.ts

```diff
@@ -23,6 +23,7 @@ import {
   SYSTEM_GROUP_ID_USER,
   SYSTEM_GROUP_ID_ADMIN,
 } from "../../../data/user";
+import { showSuccesToast } from "../../../util/toast-saved-success";
```

Comment:
Can we rename it to `showSaveSuccessToast` ?  (also with double ss at end)

Reply:
All the names for auto complete are global, so I prefer having the name identify exactly what it does.